docs/source/_json/comments_get.resp

```diff
@@ -13,6 +13,8 @@ Content-Type: application/json
       "comment_id": "1477076400000000", 
       "creation_date": "2016-10-21T19:00:00", 
       "in_reply_to": null, 
+      "is_deletable": true, 
```

Comment:
@sneridagh we decided to just "map over" existing attributes instead of cleaning them up. Are those attributes present in the same way on the discussion object?

Reply:
Had a discussion with @sneridagh on that and he argued we are mapping the attribute of the discussion class to plone.restapi, not an attribute of the object. Therefore we do not have to be too strict about our mapping-over-policy here. That somehow convinced me.